sync3/handler/connstate.go

```diff
@@ -143,16 +154,21 @@ func (s *ConnState) load(ctx context.Context, req *sync3.Request) error {
 	for _, r := range rooms {
 		s.lists.SetRoom(r)
 	}
-	s.loadPosition = initialLoadPosition
+	s.anchorLoadPosition = initialLoadPosition
 	return nil
 }
 
 // OnIncomingRequest is guaranteed to be called sequentially (it's protected by a mutex in conn.go)
 func (s *ConnState) OnIncomingRequest(ctx context.Context, cid sync3.ConnID, req *sync3.Request, isInitial bool) (*sync3.Response, error) {
-	if s.loadPosition == -1 {
+	if s.anchorLoadPosition <= 0 {
```

Comment:
Can we write down what nonpositive values mean in the comment for anchorLoadPosition? I.e. that we're using them as a sentinel to mean the conn hasn't been loaded yet